mDNSCore/mDNSEmbeddedAPI.h

```diff
@@ -53,6 +53,11 @@
 #ifndef __mDNSEmbeddedAPI_h
 #define __mDNSEmbeddedAPI_h
 
+// MinGW defines "#define interface struct" for ObjC compatibility.
```

Comment:
Is there a preprocessor predefine that's specific to MinGW? We also build in Visual Studio, which I think defines _WIN32 as well.

Reply:
Oh, I see in a later change that you're checking for __MINGW32__. Why not do that here as well?

Reply:
Yes, it's [specific](https://github.com/mirror/mingw-w64/blob/master/mingw-w64-headers/include/rpc.h) to MinGW. I will change the condition.